nunit-project-editor.nuspec

```diff
@@ -0,0 +1,23 @@
+﻿<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>nunit-project-editor</id>
+    <version>1.0</version>
+    <packageSourceUrl>https://github.com/nunit/nunit-project-editor/chocolatey/</packageSourceUrl>
+    <owners>Charlie Poole</owners>
+
+    <title>NUnit Project Editor</title>
+    <authors>Charlie Poole</authors>
+    <projectUrl>https://github.com/nunit/nunit-project-editor</projectUrl>
+    <iconUrl>http://cdn.rawgit.com/nunit/resources/master/images/icon/nunit_64.png</iconUrl>
```

Comment:
Based on that, I guess we should use 128x128 rather than 256x256 for chocolatey.